### ISSUE TITLE
Select units in formatting - W vs VA?

### ISSUE THREAD

**Issue by pdb5627** [hgrecco/pint#395]
In the default unit definitions, power is defined as follows:
[power] = [energy] / [time]
watt = joule / second = W = volt_ampere = VA

In electrical calculations, W and VA may be interchangeable from a units perspective, but when outputting results, a quantity usually is accurately represented with either W, VA, or var depending on the context. Is there any way to select which of the units get output when formatting?

**Comment by hgrecco** [comment]
There is no public api for this but might be something nice to have. How do you imagine this working?

**Comment by pdb5627** [comment]
I would like to be able to do a calculation like this:
```python
V = 115 * ureg.kV
I = (250. + 117.j)*ureg.A
S = V*I.conjugate()
P = S.real
Q = S.imag
print(S.to(ureg.MVA))
```
Current output:
```
(28.75-13.455j) megawatt
````

Desired output, either from direct or formatted string conversion:
```
(28.75-13.455j) MVA
```

It wouldn't be too bad even if I had to set the unit output string manually:
```python
MVA = ureg.MVA
MVA.symbol = 'MVA'
print(S.to(MVA))
```
The way MathCAD handles setting units in output works well. Results show with some default units, but there is a formatting block to fill that allows conversion of the result to desired units. The request to convert to the units causes the magnitude to be adjusted accordingly and sets the display of the units. I think the equivalent in pint code would be something like this, where the underscore is some flag character to indicate this type of format request:
```python
print({_MVA}.format(S))
```

As I typed this up, I poked around with this a little more, and maybe I can get what I want by defining the units myself:
```python
import pint
ureg = pint.UnitRegistry()
ureg.define('volt_ampere = 1 * volt*ampere = VA')
ureg.define('volt_ampere_reactive = 1 * volt_ampere = var')
ureg.default_format = '~'
V = 115 * ureg.kV
I = (250. + 117.j)*ureg.A
S = V*I.conjugate()
P = S.real
Q = S.imag
print(S.to(ureg.MVA))
print(P.to(ureg.MW))
print(Q.to(ureg.Mvar))
```
Output:
```
(28.75-13.455j) MVA
28.75 MW
-13.455 Mvar
```
I think the problem might be the default unit definitions file aliases `volt_ampere` to `watt` whereas I would like to be able to treat them as different units that can convert back and forth with a factor of 1.
